kieserver/image.yaml

```diff
@@ -142,6 +142,30 @@ envs:
   - name: "KIE_SERVER_DISABLE_KC_VERIFICATION"
     example: "true"
     description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."
+  - name: "KIE_KAFKA_EXT_ENABLED"
```

Comment:
prefix should be KIESERVER_
please change it here and on the jboss-kie-modules.

Reply:
prefix should be `KIE_SERVER_` , probably just typo in comment

Reply:
All KIE_SERVER_KAFKA_EXT_ now